harness/engine/profiles.go

```diff
@@ -30,6 +30,7 @@ func Profile1(index int, node *DuskNode, walletPath string) {
 	viper.Set("general.walletonly", "false")
 	viper.Set("general.safecallbacklistener", "false")
 	viper.Set("general.testharness", "true")
+	viper.Set("general.genesispath", "$HOME/go/src/github.com/dusk-network/dusk-blockchain/devnetgenesis.txt")
```

Comment:
devnetgenesis.txt seems not to be a text file? If not, could we use `dat` or `bin` extention?